pages/agent/v3/windows.md

```diff
@@ -89,6 +89,15 @@ nssm start buildkite-agent
 nssm status buildkite-agent
 ```
 
+If you'd like to change the user the buildkite-agent service runs as, you can use the same third-party tool [nssm](https://nssm.cc/) using the command-line:
+
+```
+# Change ComputerName to the name of the System under "Settings"
+# Change AccountName to the name of the Account you'd like to use
+# Change password to the password of the account you provided in the above comment
+nssm set buildkite-agent ObjectName "ComputerName\AccountName" "password"
```

Comment:
What's the `ObjectName`? It looks like it's not mentioned anywhere here. Is it meant to be replaced by another value or left as is?

Reply:
So in this case, `ObjectName` is part of the CLI syntax and not to be touched.